cpu/native/can/candev_linux.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     native_cpu
+ * @ingroup     candev_linux
```

Comment:
Not sure about this name... And technically it is in both, so I would keep the removed line.